React/Views/RCTView.m

```diff
@@ -202,6 +202,46 @@ - (BOOL)didActivateAccessibilityCustomAction:(UIAccessibilityCustomAction *)acti
 
 - (NSString *)accessibilityValue
 {
+  static dispatch_once_t onceToken;
+  static NSDictionary<NSString *, NSString *> *rolesAndStatesDescription = nil;
+
+  dispatch_once(&onceToken, ^{
+    NSString *bundlePath = [[NSBundle mainBundle]pathForResource:@"AccessibilityResources" ofType:@"bundle"];
+    NSBundle *bundle = [NSBundle bundleWithPath:bundlePath];
+
+    if (bundle) {
+      NSURL *url = [bundle URLForResource:@"Localizable" withExtension:@"strings"];
+      rolesAndStatesDescription = [NSDictionary dictionaryWithContentsOfURL:url];
```

Comment:
It looks like this method is deprecated according to https://developer.apple.com/documentation/foundation/nsdictionary/1574185-dictionarywithcontentsofurl

Can you update to use the suggested one?

Also, what is the flow for when the strings don't exist for a language? I imagine the `AccessibilityResources` bundle will exist, but the url will not. That `NSDictionary` method is safe to use with a nil url, correct?

Reply:
I'll update the method.
When the strings don't exist for a language. It will fallback to the English strings in the AccessibilityResources.bundle. The url is the one for the Localizable.strings in en.lproj. Also, if something bad happens and results in a nil url, the NSDictionary method is still safe to use.

Reply:
We'll leave the localization for the other languages to the community. The accessibility strings on the Android side have been marked as localized for some reason. https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/res/views/uimanager/values/strings_unlocalized.xml
So I think it's ok to only have the english strings for iOS for now.